utils/model_runners/gpt_model_runner.py

```diff
@@ -18,8 +21,11 @@ class GPTModelConfig:
 class GPTModelRunner(ModelRunner):
     url = "https://api.openai.com/v1/chat/completions"
     
-    def __init__(self, model_config: GPTModelConfig):
+    def __init__(self, model_config: GPTModelConfig, metrics_folder: str = None,  model_key:str = None,reset = False):
         self.config = model_config
+        self._metrics_folder = metrics_folder
+        self._model_key = model_key
+        self._reset = reset
```

Comment:
I think we will always want to "reset" (remove the old latency/tokens capture), since this is a notebook and the user could run a cell multiple times).
So I would just remove `reset` unless you have a good use case to keep it.